composer.json

```diff
@@ -20,7 +20,9 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "altapay/api-php":"^3.0"
+        "altapay/api-php":"^3.0",
+        "symfony/event-dispatcher": "^3.0",
+        "symfony/options-resolver": "^3.0"
```

Comment:
Maybe a better solution is to generate the lock fiil on php5.6 sothat the dependencies are based on it